include/copy_to_user_wrapper.h

```diff
@@ -0,0 +1,23 @@
+#ifndef COPY_TO_USER_WRAPPER_H
+#define COPY_TO_USER_WRAPPER_H
+
+struct copy_record
+{
+    void * from;
+    void * to;
+    unsigned long len;
+    void * bytes;
+    struct list_head list;
+};
+
+/*
+ * @purpose: Hook copy_to_user
+ */
+int init_copy_hook(void);
+
+/*
+ * @purpose: Unhool copy_to_user
```

Comment:
JESUS FUCKING CHRIST WHAT ARE YOU A FUCKING IDIOT WHAT THE FUCK IS UNHOOL
LEARN TO USE A KEYBOARD YOU MAGGOT